client/signup/steps/plans/index.jsx

```diff
@@ -164,13 +160,6 @@ export class PlansStep extends Component {
 					disableBloggerPlanWithNonBlogDomain={ disableBloggerPlanWithNonBlogDomain }
 					plansWithScroll={ true }
 				/>
-				{ /* The `hideFreePlan` means that we want to hide the Free Plan Info Column.
-				 * In most cases, we want to show the 'Start with Free' PlansSkipButton instead --
-				 * unless we've already selected an option that implies a paid plan.
-				 * This is in particular true for domain names. */
-				hideFreePlan && ! isDomainOnly && ! this.getDomainName() && (
-					<PlansSkipButton onClick={ this.handleFreePlanButtonClick } />
-				) }
```

Comment:
`PlansFeaturesMain` takes care of displaying the free plan banner, so this skip button is no longer needed. (https://github.com/Automattic/wp-calypso/blob/remove/plans-skip-link-in-signup/client/my-sites/plans-features-main/index.jsx#L294)